src/state/GlobalState.tsx

```diff
@@ -266,21 +266,9 @@ export class GlobalState extends PureComponent<Props> {
   }
 
   checkVersion = async () => {
-    const { t } = this.props
     const newVersion = await ipcRenderer.invoke('checkVersion')
     if (newVersion) {
-      const { response } = await ipcRenderer.invoke('openMessageBox', {
-        buttons: [t('box.yes'), t('box.no')],
-        message: t(
-          'box.appupdate.message',
-          'There is a new version of Heroic Available, do you want to update now?'
-        ),
-        title: t('box.appupdate.title', 'Update Available')
-      })
-
-      if (response === 0) {
-        ipcRenderer.send('openReleases')
-      }
+      ipcRenderer.send('updateHeroic')
```

Comment:
so you will need some kind of check here, to see if its appImage or not. probably seeing the path, if it starts with /tmp/ will return true if appImage. But I don't know about installed appImages, maybe they will go to /opt, then it won't work.

Reply:
Hmmm...maybe check if it's not an app image? like, appimages aren't on /usr, they are on home or the hdd